WooCommerce/Classes/ViewRelated/Products/Variations/Add Attributes/AddAttributeViewController.swift

```diff
@@ -101,12 +92,13 @@ private extension AddAttributeViewController {
             case .initialized:
                 break
             case .syncing:
-                self?.displayGhostTableView()
+                self?.displayGhostContent()
             case .failed:
-                self?.removeGhostTableView()
+                self?.removeGhostContent()
                 self?.displaySyncingErrorNotice()
             case .synced:
-                self?.removeGhostTableView()
+                self?.removeGhostContent()
+                self?.tableView.reloadData()
```

Comment:
Nit: should we reload data first before removing the loading animation - just in case the data takes a bit more time to load?

Reply:
Good point, changed :)